library/dellemc/idrac/dellemc_idrac_firmware.py

```diff
@@ -22,84 +21,171 @@
 DOCUMENTATION = r'''
 ---
 module: dellemc_idrac_firmware
-short_description: Firmware update from a repository on a network share (CIFS, NFS).
+short_description: Firmware update using a repository hosted on a remote network share (CIFS, NFS) or a URL (HTTP, HTTPS, FTP)
```

Comment:
Either 'remote network share' or existing 'network share' consistently through out the documentation, I can a mix a both in the entire documentation section